circuits/cpp/barretenberg/cpp/src/barretenberg/honk/proof_system/prover_library.hpp

```diff
@@ -1,17 +0,0 @@
-#pragma once
```

Comment:
I was able to remove the prover_library because only the Instance was using it. We could do the same with the grand_product_library if we sunset StandardHonk or it is updated to suport instances.